src/features/metadata-instance-editor/Instance.js

```diff
@@ -134,10 +134,10 @@ class Instance extends React.PureComponent<Props, State> {
     componentDidUpdate() {
         const element = this.collapsibleRef.current;
         if (element && this.state.shouldConfirmRemove) {
-            scrollIntoViewIfNeeded(element, false, {
-                centerIfNeeded: false,
-                duration: 320,
-                easing: 'easeInOut',
+            scrollIntoViewIfNeeded(element, {
```

Comment:
The transition behave the same to my eyes. Not sure if we must of the duration? If so, we will need to do introduce a callback that handles that which will complicate things.